package.json

```diff
@@ -14,9 +14,10 @@
     "postinstall": "npm run precompile",
     "precompile": "node parseGitCommit.js",
     "compile": "tsc && cross-os postcompile",
```

Comment:
Should the `compile` script delete the `./dist` folder before compiling? Or will it be ignored and overwritten anyways because `incremental` isn't enabled anymore?

Reply:
I didn't think this was necessary - since indeed this should overwrite all generated files - but I thought about it some more after seeing your comment and one potential issue is that deleted files will stick around in the `dist` folder. It shouldn't hurt in terms of causing errors, but it does mean there may be files unintentionally published/packaged from the `dist` folder. So I'll see about deleting the folder before publishing